Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
```

Comment:
Unless ` stable ` needs a patch, or a unique dependency that ` devel ` doesn't, you can just kill the ` stable do ` block and implement those lines under the ` homepage ` above.